mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -22,7 +22,8 @@ CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
 	METHOD method_5442 isInvEmpty ()Z
 	METHOD method_5443 canPlayerUseInv (Lnet/minecraft/class_1657;)Z
 		ARG 1 player
-	METHOD method_5444 getInvMaxStackAmount ()I
+	METHOD method_5444 getMaxCountPerStack ()I
```

Comment:
```suggestion
	METHOD method_5444 getInvStackMaxAmount ()I
```

Reply:
I think `getMaxCountPerStack` is clearer than `getInvStackMaxCount`.

Reply:
`per` sounds to be as if this method is stack/slot aware, but it's not.

Reply:
I vote for `getMaxCount/AmountPerStack` tho.

Reply:
`getInvStackMaxCount` does not resolve the initial issue, being that you could confuse it as meaning "max count of stacks in the inventory".

Reply:
changing per to of may be clearer, i guess.

Reply:
```suggestion
	METHOD method_5444 getMaxCountOfStack ()I
```

Reply:
`getMaxCountOfStacks` ?

Reply:
`getMaxCountOfStacks` sounds like something that returns how many stacks can be in the inventory.

Reply:
True, it's not much of an improvement.

Reply:
Then just `getMaxCountOfStack` so you know it applies to one stack rather than all the stacks